app/assets/stylesheets/application.scss

```diff
@@ -177,13 +177,18 @@ body {
       }
     }
 
+    .diy-header {
+      max-width: 450px;
+    }
+
     .diy-subheader {
       font-weight: 200;
-      max-width: 391px;
+      max-width: 450px;
```

Comment:
test copy is much longer and needed more space. original copy works fine with this too